product/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM maven:3.8.6-eclipse-temurin-17-alpine as build
+RUN mkdir -p /build
+WORKDIR /build
+
+COPY ./ ./
+
+RUN mvn clean package -DskipTests=true
```

Comment:
I think the maven command should not add here because the build docker image will execute after the build phase is successful. Just copying the product package is enough

Reply:
Yes, as your recommend, i removed it from the build image. Pls review it again for me @hvantran  👍